File: solutions/1329-minimum-cost-to-move-chips-to-the-same-position/solution.py
class Solution(object):
    def minCostToMoveChips(self, position):
        """
        :type position: List[int]
        :rtype: int
        """
        e=0
        o=0
        for i in position:
            if(i&1)==1:
                o+=1;
            else:
                e+=1;
        return o if o<e else e

File: solutions/1329-minimum-cost-to-move-chips-to-the-same-position/solution.java
class Solution {
    public int minCostToMoveChips(int[] position) {
    int e=0,o=0;
    for(int i=0;i<position.length;i++)
    {
        if((position[i]&1)==1)
            o++;
        else
            e++;
    }
    return o<e?o:e;
    }
}
